### PR TITLE
Update Safari instructions

### PR DESCRIPTION
**Pull request by ngyikp**
Fixes https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/2614

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,12 +142,12 @@ All that is asked is that you have at least one previous contribution to RES.
 
 ##### Building in Safari (assumes Mac)
 
-  1. Open the `Preferences` by going to `Safari->Preferences` or pressing `⌘`, then go to `Advanced` and check the checkbox for `Show develop menu in menu bar`.
-  2. Navigate to `Develop->Show Extension Builder` to open the extensions menu. Add a new extension by pressing the `+` in the bottom left and choosing `Add extension`.
+  1. Open the `Preferences` by going to `Safari->Preferences` or pressing `⌘,` then go to `Advanced` and check the checkbox for `Show Develop menu in menu bar`.
```
Comment (13steinj) [inline README.md]: The comma should be outside the \`, not inside it (unless I misunderstand my grammar rules).

Comment (ngyikp) [inline README.md]: The keyboard shortcut to open Safari Preferences *is* pressing the Command
and the comma key

http://www.bu.edu/tech/files/2009/10/safari-preferences2.gif

Comment (13steinj) [inline README.md]: Ah, never mind me then.

**Comment by mc10** [comment]
Thanks!

**Pull request merged**